pkgs/development/libraries/modern-cpp-kafka/default.nix

```diff
@@ -0,0 +1,48 @@
+{ rdkafka, stdenv, lib, fetchFromGitHub, cmake, zlib, breakpointHook, boost, gtest, rapidjson, pkg-config }:
+
+stdenv.mkDerivation rec {
+  pname = "modern-cpp-kafka";
+  version = "v2023.03.07";
+
+  src = fetchFromGitHub {
+    repo = "modern-cpp-kafka";
+    # owner = "morganstanley";
+    # rev = version;
+    # hash = "sha256-7hkwM1YbveQpDRqwMZ3MXM88LTwlAT7uB8NL0t409To=";
+    #
+    # Patch build to avoid overwriting include path
+    # PR: https://github.com/morganstanley/modern-cpp-kafka/pull/221
+    owner = "ditsuke";
+    rev = "build/avoid-overwriting-include-path";
```

Comment:
Can you use the upstream version of this project anyway and add your patches via the `patches` ? Or, since your PR is still very young, maybe just wait a bit and see if they accept it.

Reply:
The build won't work without it, but I've added it as a patch for now.